frontend/javascripts/libs/mjs.js

```diff
@@ -86,7 +87,7 @@ M4x4.transformPoints = function transformPoints(
   if (r == null) {
     r = new Float32Array(points.length);
   }
-  for (let i = 0; i < points.length; i += 3) {
+  for (let i = 0; i < points.length; i = i + 3) {
```

Comment:
Was there any specific reason for this?

Reply:
Yeah, I came accross the comment in line 56 "// DO NOT CHANGE to let compound assignment (+=) as V8 cannot optimize this" because my IDE optimized the assignment in line 57 to `+=`.

Therefore I thought that if the comment is correct, the compound assignment in line 89 should is wrong. That why I changed this. I personally do not know why this comment was added, but I can look it up:
Here we go: This is where this comment/change was introduced 
https://github.com/scalableminds/webknossos/issues/1857#issuecomment-305641037

Reply:
However, I did not do any performance testing on this

Reply:
so can I merge this?

Reply:
Ah, interesting. I assume that's not really up to date anymore, since the comment is 4 years old and already then stated:

> So it looks like the new V8 optimizing compiler Turbofan will be turned on by default in the next chrome version (59), which doesn't suffer from those deoptimization issues.

Maybe just undo your change :)